numpy/core/tests/test_multiarray.py

```diff
@@ -2907,7 +2907,8 @@ def norm(result):
                                      "__array_ufunc__")
                     else:
                         if (isinstance(obj, np.ndarray) and
-                                not hasattr(obj, "__array_ufunc__")):
+                            (obj.__class__.__array_ufunc__ is
```

Comment:
OK, replaced it with `type(obj)`.  Also raised https://github.com/numpy/numpy/issues/8877 for the OT topic.